scaife_viewer/settings.py

```diff
@@ -11,7 +11,7 @@
 TRACING_ENABLED = bool(int(os.environ.get("TRACING_ENABLED", not DEBUG)))
 
 DATABASES = {
-    "default": dj_database_url.config(default="postgres://localhost/scaife-viewer")
+    "default": dj_database_url.config(default=os.environ.get("DJ_DATABASE_URL","postgres://localhost/scaife-viewer"))
```

Comment:
This should not be needed. `dj_database_url.config` is already looking for a `DATABASE_URL` and if it fails that will use the `default=` kwarg passed in.

Reply:
Ah ok! I didn't know it was already looking for `DATABASE_URL`.